logical/framework/path.go

```diff
@@ -52,6 +58,13 @@ type Path struct {
 	// whereas all fields are available in the Write operation.
 	Fields map[string]*FieldSchema
 
+	// Operations is the set of operations supported and the associated OperationsHandler.
+	//
+	// If both Create and Update operations are present, documentation and examples from
+	// the Update definition will be used. Similarly if both Read and List are present,
+	// Read will be used for documentation.
```

Comment:
What about cases where the allowed values differ between Create/Update?

Reply:
It's tricky. POST/PUT are the split in OpenAPI so that's the only formal way to output different descriptions. But our handling is not based on whether the method is POST/PUT, so publishing differently under those headings is probably going to cause confusion.

We could look at adding metadata `x-vault...` or multiple examples, but I think that is not going to very visible.

Maybe we do simply publish both POST/PUT when create is supported and defined explicitly, and have a big explanation about how if a single path has both methods, here's what it means in Vault.  I'd probably go that route if we deem it necessary.

Reply:
Okay. This isn't common now, so I guess let's cross that bridge when we come to it.